ui/admin/app/styles/app.scss

```diff
@@ -597,3 +597,68 @@
     padding: 0 sizing.rems(l);
   }
 }
+
+.ordered-series-diagram {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+
+  .ordered-series-diagram-item {
+    margin-right: sizing.rems(m);
+    display: grid;
+    grid-template-columns: repeat(auto-fit, 100px);
+    grid-template-rows: repeat(2, 1fr);
+    gap: sizing.rems(xs) 0;
+    grid-template-areas:
+      'icon arrow'
+      'description ...';
+
+    .hds-icon-tile {
+      grid-area: icon;
+      justify-self: center;
+    }
+
+    > span {
+      text-align: center;
+      grid-area: description;
+    }
+
+    .flight-icon-arrow-right {
+      grid-area: arrow;
+      place-self: center;
+    }
+  }
+
+  .ordered-series-diagram-group {
+    border-radius: sizing.rems(xs);
+    display: grid;
+    grid-template-areas:
+      'title title'
+      'primary secondary';
+
+    > span {
+      font-size: sizing.rems(s);
+      color: var(--token-color-foreground-highlight);
+      padding-bottom: sizing.rems(s);
+      padding-top: sizing.rems(xxs);
+      grid-area: title;
+      place-self: center;
+    }
+
+    .ordered-series-diagram-item:first-child {
+      grid-area: primary;
+    }
+
+    .ordered-series-diagram-item:last-child {
+      grid-area: secondary;
+    }
+  }
+}
+
+.highlight {
+  background-color: var(--token-color-palette-purple-100);
```

Comment:
In the wireframe the background color matches. `--token-color-palette-purple-500`

Reply:
Let's use `--token-color-surface-highlight` for this.  We should use the semantic tokens rather than the palette tokens, since semantic tokens are mapped for dark mode.

Reply:
Yes, even better. `--token-color-foreground-highlight-high-contrast` maps to `--token-color-palette-purple-500` which is what the wireframe is using. `--token-color-surface-highlight` maps to `--token-color-palette-purple-50` which is a darker purple. I think `--token-color-foreground-highlight-high-contrast` would be the semantic token we need here.

Reply:
@cameronperera it is actually --token-color-foreground-highlight.  But I think @joshuaogle has the opacity reduced 40%.  I don't think `lighten()` works on HDS tokens, but I will try.

Reply:
We don't want to modify the colors, since that wouldn't work for dark mode.  Let's find the semantic token that most closely matches and then review.